code/modules/jobs/job_types/roguetown/garrison/citywatch.dm

```diff
@@ -45,27 +45,22 @@
 	ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
 
 /datum/outfit/job/roguetown/citywatch/proc/assign_skills(mob/living/carbon/human/guard)
-	guard.mind.adjust_skillrank(/datum/skill/combat/maces, pick(3,4), TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/unarmed, 4, TRUE) 
-	guard.mind.adjust_skillrank(/datum/skill/combat/swords, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/bows, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/combat/crossbows, 2, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/misc/climbing, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/misc/sneaking, 3, TRUE) 
+	guard.mind.adjust_skillrank(/datum/skill/combat/maces, 4, TRUE)
+   	guard.mind.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+	guard.mind.adjust_skillrank(/datum/skill/combat/bows, pick(3), TRUE)
+  	guard.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+	guard.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
+	guard.mind.adjust_skillrank(/datum/skill/combat/polearms, 2, TRUE)
+	guard.mind.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
 	guard.mind.adjust_skillrank(/datum/skill/misc/swimming, 2, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/misc/treatment, 1, TRUE)
-	guard.mind.adjust_skillrank(/datum/skill/misc/riding, 2, TRUE)
-	guard.change_stat("strength", 1)
-	guard.change_stat("perception", 2) 
-	guard.change_stat("constitution", 1)
-	guard.change_stat("endurance", 1)
-	guard.change_stat("speed", 1)
+	guard.mind.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
```

Comment:
Climbing is a key stat for watchmen to be able to follow after rogues/bandits who can climb. should be 3

Reply:
Fair enough 👍

Reply:
if rogues and bandits have climbing 3 it doesnt mean that now EVERYONE should get same skill to chase them

Reply:
> if rogues and bandits have climbing 3 it doesnt mean that now EVERYONE should get same skill to chase them

they always had climbing 3, all the way back in blackstone it was expressly not changed for the same reason lol